ambry-frontend/src/main/java/com/github/ambry/frontend/PostBlobHandler.java

```diff
@@ -154,6 +154,7 @@ private void start() {
         // Start the callback chain by parsing blob info headers and performing request security processing.
         BlobInfo blobInfo = getBlobInfoFromRequest();
         checkUploadRequirements(blobInfo.getBlobProperties());
+        restRequest.getMetricsTracker().setBytesTransferred(restRequest.getBytesReceived());
```

Comment:
Same comment as above. I think this might have to be called in `securityProcessResponseCallback()`